src/utils/selectors.ts

```diff
@@ -122,6 +121,8 @@ export const chatUsersSelector = (state: RootState) =>
 export const venueSelector = (state: RootState) =>
```

Comment:
[nit] a line of whitespace between these would make them easier to read (since they break across the line)